src/trials.jl

```diff
@@ -168,37 +184,44 @@ function ratio(a::TrialEstimate, b::TrialEstimate)
     ttol = max(params(a).time_tolerance, params(b).time_tolerance)
     mtol = max(params(a).memory_tolerance, params(b).memory_tolerance)
     p = Parameters(params(a); time_tolerance = ttol, memory_tolerance = mtol)
-    return TrialRatio(p, ratio(time(a), time(b)), ratio(gctime(a), gctime(b)),
-                      ratio(memory(a), memory(b)), ratio(allocs(a), allocs(b)))
+    return TrialRatio(p, ratio(realtime(a), realtime(b)), ratio(cputime(a), cputime(b)),
+                      ratio(gctime(a), gctime(b)), ratio(memory(a), memory(b)),
+                      ratio(allocs(a), allocs(b)))
 end
 
-gcratio(t::TrialEstimate) =  ratio(gctime(t), time(t))
+gcratio(t::TrialEstimate) =  ratio(gctime(t), realtime(t))
+cpuratio(t::TrialEstimate) =  ratio(cputime(t), realtime(t))
 
 ##################
 # TrialJudgement #
 ##################
 
 struct TrialJudgement
     ratio::TrialRatio
-    time::Symbol
+    realtime::Symbol
+    cputime::Symbol
     memory::Symbol
 end
 
 function TrialJudgement(r::TrialRatio)
     ttol = params(r).time_tolerance
     mtol = params(r).memory_tolerance
-    return TrialJudgement(r, judge(time(r), ttol), judge(memory(r), mtol))
+    return TrialJudgement(r, judge(realtime(r), ttol), judge(cputime(r), ttol), judge(memory(r), mtol))
```

Comment:
@jrevels I think using the same tolerance for `cputime` and `realtime` should be ok, right?

Reply:
Sure, the tolerance is a fudge number anyway.